src/main/kotlin/no/nav/familie/ef/søknad/api/dto/søknadsdialog/SøknadBarnetilsynDto.kt

```diff
@@ -15,11 +15,12 @@ data class SøknadBarnetilsynDto(
     val dokumentasjonsbehov: List<Dokumentasjonsbehov>,
     val locale: String = "nb",
     val skalBehandlesINySaksbehandling: Boolean = false,
-    val datoPåbegyntSøknad: LocalDate? = null
+    val datoPåbegyntSøknad: LocalDate? = null,
 )
 
 data class SøknadBarnetilsynGjenbrukDto(
     val sivilstatus: SivilstatusTilGjenbruk,
     val medlemskap: Medlemskap,
     val bosituasjon: Bosituasjon,
+    val person: PersonTilGjenbruk,
```

Comment:
Hva benyttes denne til? 🤔

Reply:
Denne tar vekk info om søkeren ved gjenbruk, da det skal hentes fra PDL. Det var enkleste måte å sikre at de ikke gikk i beina på hverandre, for det ble litt krøll i frontend når data skulle lastes inn i state